backend/src/main/java/com/chinguv45tier3team53/tripplannerrest/controller/TripController.java

```diff
@@ -0,0 +1,41 @@
+package com.chinguv45tier3team53.tripplannerrest.controller;
+
+import com.chinguv45tier3team53.tripplannerrest.entity.Trip;
+import com.chinguv45tier3team53.tripplannerrest.repository.TripRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/trips")
+public class TripController {
+    @Autowired
+    private TripRepository tripRepository;
+    @GetMapping
+    public List<Trip> getAllTrips(){
+        return tripRepository.findAll();
```

Comment:
best to make a service class and calling that rather than making direct calls from controller to repository